### DISCUSSION TITLE
Version information in "UI" section

### DISCUSSION THREAD

**Discussion by mikelupe** [ZL-Audio/ZLEqualizer#55] [Ideas]
It would be nice to see the version of the plugin in the "UI" section.

For your official releases your defined version, for self-built maybe the git commit with date, something like surgeXT does.

**Comment by zsliu98** [comment]
Added in the latest commit. The current version and the hash of the latest commit are shown at the bottom-left corner of the UI setting panel. The date is not included as it might confuse other users.

**Reply by mikelupe** [reply]
Makes sense, great. 

It'll facilitate testing, as I'm checking on several machines with some divergent OS configs.
